src/components/pages/order/Main/Basket/Basket.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+import { formatPrice } from "../../../../../utils/maths";
+import BasketBody from "./BasketBody";
+import Footer from "./Footer";
+import Total from "./Total";
+
+function Basket() {
+  return (
+    <BasketStyled>
+      <Total AmountToPay={formatPrice(0)} />
```

Comment:
```suggestion
      <Total amountToPay={formatPrice(0)} />
```
Ce n'est pas un composant là non plus.

Reply:
Je corrige ça ...